README.rst

```diff
@@ -181,6 +184,13 @@ Project resources
 - `Changelog <https://github.com/mopidy/mopidy-local/releases>`_
 
 
+Build locally and run
```

Comment:
We have development envinronment guidelines at https://docs.mopidy.com/en/latest/devenv/#working-on-extensions and this doesn't match that.

Reply:
thanks! I was looking for such doc directly in the plugin's readme .. may I suggest to add a link from there?

Reply:
Yes, it seems we can never have enough links back to the docs. We might as well add it to https://github.com/mopidy/cookiecutter-mopidy-ext at some point

Reply:
by the way, this is an amazing doc that you have, this is rare enough to be reported! really, congrats
(and shame on me for not having seen it before)